admin/users/views.py

```diff
@@ -132,47 +124,30 @@ def get_object(self, queryset=None):
 class ResetPasswordView(OSFAdmin, FormView):
     form_class = EmailResetForm
     template_name = 'users/reset.html'
-
-    def get(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).get(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
-
-    def post(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).post(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
+    context_object_name = 'user'
 
     def get_context_data(self, **kwargs):
         user = User.load(self.kwargs.get('guid'))
         try:
             self.initial.setdefault('emails', [(r, r) for r in user.emails])
         except AttributeError:
```

Comment:
409 is more of a conflict of the database from what I've read. The fact that it isn't included in the Django framework but is in the DRF is interesting. And 400 is more for malformed requests. I think a 'not found' describes the: "I didn't find the original user with that email address." Which I can certainly update this to say.